.kitchen.yml

```diff
@@ -13,19 +13,32 @@ platforms:
   - name: ubuntu-12.04
   - name: ubuntu-14.04
   - name: centos-6.5
+  - name: centos-7.0
 
 suites:
   - name: default
     run_list:
       - recipe[mysql-multi::default]
     attributes:
+      mysql-multi:
+        master_ip: '192.168.0.21'
+        slave_ip: ['192.168.0.22']
+        server_repl_password: 'VeryBadReplPasswd'
 
   - name: master
     run_list:
       - recipe[mysql-multi::mysql_master]
     attributes:
+      mysql-multi:
+        master_ip: '192.168.0.21'
+        slave_ip: ['192.168.0.22']
+        server_repl_password: 'VeryBadReplPasswd'
 
   - name: slave
     run_list:
       - recipe[mysql-multi::mysql_slave]
     attributes:
+      mysql-multi:
+        master_ip: '192.168.0.21'
+        slave_ip: ['192.168.0.22']
+        server_repl_password: 'VeryBadReplPasswd'
```

Comment:
I think we can add all of this attributes in the `provisioner` section (to prevent duplication)